packages/in_app_purchase_tizen/lib/src/in_app_purchase_tizen_platform.dart

```diff
@@ -0,0 +1,160 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+import 'package:in_app_purchase_platform_interface/in_app_purchase_platform_interface.dart';
+
+import '../billing_client_wrappers.dart';
+
+const String kPurchaseErrorCode = 'purchase_error';
+
+const String kIAPSource = 'samsung_checkout';
+
+late String appId;
```

Comment:
If you define 'appId' as global variable, please name it 'kAppId', and if it is private, name it '_kAppId'.
And why not define 'appId' in class 'InAppPurchaseTizenPlatform'?

Reply:
fixed